app/plugins/privileges/privileges.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+angular.module('privileges', ['sql'])
+  .factory('UserListService', function(SQLQuery, queryResultToObjects, $q) {
+    var UserListService = {
+      deferred: $q.defer()
+    };
+
+    var stmt = 'select name, superuser from sys.users order by name';
```

Comment:
Would it be possible to format this (and other sql statments) with capitalisation on keywords? `SELECT name, superuser FROM sys.users ORDER BY name` for example, just to make it a little easier to read